objects/Fan-MadeAccessories.aa8b38/CleanUpHelper.26cf4b.ttslua

```diff
@@ -29,24 +30,21 @@ local IGNORE_TAG = "CleanUpHelper_ignore"
 -- colors and order for following tables
 local COLORS = { "White"; "Orange"; "Green"; "Red"; "Agenda" }
 
--- counter GUIDS (4x damage, 4x sanity and 4x resource)
-local TOKEN_GUIDS = {
+-- counter GUIDS (4x damage and 4x horror)
+local DAMAGE_HORROR_GUIDS = {
   "eb08d6"; "e64eec"; "1f5a0a"; "591a45";
   "468e88"; "0257d9"; "7b5729"; "beb964";
-  "4406f0"; "816d84"; "cd15ac"; "a4b60d"
 }
 
--- default values (4x damage, 4x horror, 4x resources)
-local DEFAULT_VALUES = {
-  0; 0; 0; 0;
-  0; 0; 0; 0;
-  5; 5; 5; 5
-}
+local RESET_VALUES = {}
 
-local PLAYERMAT_GUIDS = { "8b081b"; "bd0ff4"; "383d8b"; "0840d5" }
-local TRACKER_GUIDS   = { "e598c2"; "b4a5f7"; "af7ed7"; "e74881" }
-local CLUE_GUIDS      = { "d86b7c"; "1769ed"; "032300"; "37be78" }
-local TRASHCAN_GUIDS  = { "147e80"; "f7b6c8"; "5f896a"; "4b8594"; "70b9f6" }
+-- GUIDS of objects (in order of ownership relating to 'COLORS')
```

Comment:
List the colors here, I find different people have different natural ordering for colors

Reply:
It's listed in LOC #31